README.md

```diff
@@ -21,3 +21,11 @@ $ python weather_check.py 15 51.123456 -1.123456 --secret SomeSecretKeyHere!
 2017-01-06 09:58:17,943 Starting new HTTPS connection (1): api.darksky.net
 2017-01-06 09:58:18,350 You don't need to worry about salt tonight.
 ```
+
+## Prerequisites:
+
+This script requires Python and the argparse, logging & requests Python modules. For Ubuntu only the requests module is not included with the basic Python pacakges.  It can be installed by:
+
+```
+sudo apt install python python-requests
```

Comment:
Its usually preferable to use the Python package Index to install python libraries via the `pip` tool (access to many more packages and versions, less likely to cause issues with other programs etc.). ([source](https://askubuntu.com/questions/431780/apt-get-install-vs-pip-install#431811))

The README should point to the python install guide, followed by ```pip install -r requirements.txt``` or similar (Which I didn't add because I mistakenly thought requests was part of the standard library).  Note also that Ubuntu and _most_ other linux distributions ship with some version of python by default ([source](http://unix.stackexchange.com/a/24808))